chucks-pick3-web/angular-app/src/app/components/info.component/info.component.html

```diff
@@ -2,5 +2,5 @@
   <h1><b>Requirements:</b></h1>
 </div>
 <div class="what-you-need">
-  <img src="/assets/images/Info page -11.png" alt="IPhone">
+  <img src="src/app/assets/images/Info page -11.png" alt="IPhone">
```

Comment:
Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

Reply:
reverted back to assets/images.